From 20e4c529458fa42ef6651a0042e3955723ee20c2 Mon Sep 17 00:00:00 2001
From: Mike Pall <mike>
Date: Thu, 10 Jan 2019 12:28:24 +0100
Subject: [PATCH 55/72] DynASM/PPC: Fix shadowed variable.

Cleanup only, bug cannot trigger.
Thanks to Domingo Alvarez Duarte.
---
 dynasm/dasm_ppc.lua | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/dynasm/dasm_ppc.lua b/dynasm/dasm_ppc.lua
index 4e1656e..77031fb 100644
--- a/dynasm/dasm_ppc.lua
+++ b/dynasm/dasm_ppc.lua
@@ -1056,9 +1056,9 @@ map_op[".template__"] = function(params, template, nparams)
     elseif p == "M" then
       op = op + parse_shiftmask(params[n], false); n = n + 1
     elseif p == "J" or p == "K" then
-      local mode, n, s = parse_label(params[n], false)
-      if p == "K" then n = n + 2048 end
-      waction("REL_"..mode, n, s, 1)
+      local mode, m, s = parse_label(params[n], false)
+      if p == "K" then m = m + 2048 end
+      waction("REL_"..mode, m, s, 1)
       n = n + 1
     elseif p == "0" then
       if band(shr(op, rs), 31) == 0 then werror("cannot use r0") end
-- 
2.20.1

